refresh/templates/extensions/PushNotificationsExtension.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import BluemixPushNotifications
+import CloudFoundryConfig
+
+extension PushNotifications {
+
+   public convenience init(service: PushNotificationsService) {
+
+        let regionVar = "<%- service.region %>"
+        var region: String
+
+        regionVar.isEmpty ? (region = PushNotifications.Region.US_SOUTH) : (region = regionVar)
+
+        self.init(
+            bluemixRegion: region,
+            bluemixAppGuid: "<%- service.guid %>", 
+            bluemixAppSecret: "<%- service.secret %>" )
```

Comment:
This looks a little strange to me, I wasn't expecting to see template tags in this file. I would have expected these values to be available from the `service` parameter, is that incorrect?

I tried to check by looking up the `PushNotificationsService` in CloudConfiguration (or @jkingoliver 's fork) but I couldn't find it. I did check the service credentials Bluemix gives and it does seem to supply `appGuid`, `url`, `admin_url`, `appSecret` and `clientSecret`.

So I guess this was a mistake and these are not supposed to be templated?